### PR TITLE
Hide autosuggest visually instead of not rendering it when it's not open

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
DDFSOEG-346

#### Description
This PR addresses a failing accessibility test in the dpl-cms once dpl-react is set into drupal. The test was complaining that our search header label for autosuggest was referring to a non-existing DOM element, because we didn't render autosuggest at all unless it should be visible. Now, we instead just hide it visually.

#### Screenshot of the result
image

#### Checklist
- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions
n/a

### CODE REVIEW

File: src/components/autosuggest/autosuggest.tsx [hunk 1]
```diff
@@ -46,7 +47,7 @@ export const Autosuggest: React.FC<AutosuggestProps> = ({
       {/* eslint-disable react/jsx-props-no-spreading */}
       {/* The downshift combobox works this way by design */}
       <ul
-        className="autosuggest pb-16"
+        className={`autosuggest pb-16 ${clsx({ "hide-visually": !isOpen })}`}
         {...getMenuProps()}
         style={!isOpen ? { display: "none" } : {}}
```
Comment (kasperg) [inline src/components/autosuggest/autosuggest.tsx:52]: Please consider whether this change is actually needed.

At the moment we already hide the autosuggest if it is not open by setting `display: none`.

File: src/apps/search-header/search-header.test.ts [hunk 2]
```diff
@@ -38,7 +38,7 @@ describe("Search header app", () => {
 
   it("Doesn't show suggestions before 3 characters are typed", () => {
     cy.get(".header__menu-search-input").focus().type("ha");
-    cy.get(".autosuggest").should("not.exist");
+    cy.get(".autosuggest").should("not.be.visible");
```
Comment (kasperg) [inline src/apps/search-header/search-header.test.ts:41]: :+1:

File: src/components/autosuggest/autosuggest.tsx (hunk 1 above)
Comment (Adamik10) [reply src/components/autosuggest/autosuggest.tsx]: Oh my, I totally should have noticed that 🙈

**Review by kasperg — APPROVED** [review]
:+1: LGTM

**Review by sinejespersen — APPROVED** [review]

**Pull request merged**